docset/windows/remotedesktop/New-RDSessionDeployment.md

```diff
@@ -30,6 +30,9 @@ New-RDSessionDeployment [-ConnectionBroker] <String> [-SessionHost] <String[]> [
  [<CommonParameters>]
 ```
 
+> [!IMPORTANT]
```

Comment:
Because PowerShell itself doesn't support markdown, can you change this to:

Important: \<text\>